prime-dotnet-webapi/ViewModels/SiteRegistration/SiteUpdateModel.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+using Prime.Models;
+
+namespace Prime.ViewModels
+{
+    public class SiteUpdateModel
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public PhysicalAddress PhysicalAddress { get; set; }
+
+        public int? AdministratorPharmaNetId { get; set; }
+
+        public Party AdministratorPharmaNet { get; set; }
+
+        public int? PrivacyOfficerId { get; set; }
+
+        public Party PrivacyOfficer { get; set; }
+
+        public int? TechnicalSupportId { get; set; }
+
+        public Party TechnicalSupport { get; set; }
+
+        public int? OrganizationTypeCode { get; set; }
+
+        [JsonIgnore]
+        public OrganizationType OrganizationType { get; set; }
```

Comment:
If this is JsonIgnored, the FE can't send this up, right? Do we need this on the update model?